src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4765,8 +4765,6 @@ internal static SessionStateAliasEntry[] BuiltInAliases
 
         internal const string DefaultPromptFunctionText = @"
 ""PS $($executionContext.SessionState.Path.CurrentLocation)$('>' * ($nestedPromptLevel + 1)) "";
```

Comment:
```suggestion
""PS $($executionContext.SessionState.Path.CurrentLocation)$('>' * ($nestedPromptLevel + 1)) "";
# .Link
# https://go.microsoft.com/fwlink/?LinkId=2136843
```

Reply:
Added new link

Reply:
This is not the correct link.  `HELP PROMPT` and `HELP PROMPT -O` should display the same content.